src/main/kotlin/io/github/mojira/arisa/modules/RemoveVersionModule.kt

```diff
@@ -0,0 +1,39 @@
+package io.github.mojira.arisa.modules
+
+import arrow.core.Either
+import arrow.core.extensions.fx
+import io.github.mojira.arisa.domain.Issue
+import java.time.Instant
+
+class RemoveVersionModule : Module {
+    override fun invoke(issue: Issue, lastRun: Instant): Either<ModuleError, ModuleResponse> = with(issue) {
+        Either.fx {
+            val addedVersions = getExtraVersionsLatelyAddedByNonVolunteers(lastRun)
+            val removeAddedVersions = affectedVersions
+                .filter { it.id in addedVersions }
+                .map { it.remove }
+            assertNotEmpty(removeAddedVersions).bind()
+            removeAddedVersions.forEach(::run)
+        }
+    }
+
+    private fun Issue.getExtraVersionsLatelyAddedByNonVolunteers(lastRun: Instant): List<String> =
+        if (created.isAfter(lastRun)) {
+            if (resolution == "Unresolved" || isVolunteer(reporter?.getGroups?.invoke())) {

```

Comment:
You don't need to check the resolution here. The module doesn't see any tickets that do not match the resolution as defined in the json config.

Reply:
👍